lms/djangoapps/discussion/views.py

```diff
@@ -200,7 +200,7 @@ def get_pages_parameters(results):
     query_params['num_pages'] = num_pages if returning_threads else 1
     query_params['corrected_text'] = corrected_text
 
-    return threads, query_params
+    return returning_threads, query_params
```

Comment:
we should rename `returning_threads` to something useful i.e `filtered_threads` ?